client/components/ReviewsWidget.jsx

```diff
@@ -1,20 +1,167 @@
+          },
+          {
+              "id": 67842,
+              "url": "https://images.unsplash.com/photo-1544376664-80b17f09d399?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1525&q=80"
+          }
+      ]
+  },
+  {
+      "review_id": 91793,
+      "rating": 3,
+      "summary": "Quis unde maxime ut voluptas perspiciatis et ex exercitationem.",
+      "recommend": true,
+      "response": null,
+      "body": "Ut omnis consequatur. Provident dolorem at est harum beatae natus omnis in est. Aliquam est iste neque magni. Temporibus debitis ex sed perspiciatis tempora voluptatem minima quasi dolores.",
+      "date": "2020-08-20T00:00:00.000Z",
+      "reviewer_name": "Abby.Batz68",
+      "helpfulness": 9,
+      "photos": [
+          {
+              "id": 67843,
+              "url": "https://images.unsplash.com/photo-1517278322228-3fe7a86cf6f0?ixlib=rb-1.2.1&auto=format&fit=crop&w=1567&q=80"
+          },
+          {
+              "id": 67844,
+              "url": "https://images.unsplash.com/photo-1486025402772-bc179c8dfb0e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
+          }
+      ]
+  },
+  {
+      "review_id": 91790,
+      "rating": 4,
+      "summary": "Quis voluptatibus sed et.",
+      "recommend": true,
+      "response": null,
+      "body": "Aut iusto quos excepturi nobis saepe exercitationem illo et veritatis. Atque dolores deserunt repudiandae. Placeat commodi perspiciatis consequuntur in. Minus et et. Aut error est accusamus corporis quo. Quae qui reprehenderit et nisi omnis illo neque cum.",
+      "date": "2020-04-02T00:00:00.000Z",
+      "reviewer_name": "Nelda76",
+      "helpfulness": 13,
+      "photos": [
+          {
+              "id": 67851,
+              "url": "https://images.unsplash.com/photo-1546367791-e7447b431084?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
+          },
+          {
+              "id": 67852,
+              "url": "https://images.unsplash.com/photo-1520904549193-5ab0027b3fa6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80"
+          }
+      ]
+  }
+];
 
 export class ReviewsWidget extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      filteredReviews: dummyData,
+      productReviews: dummyData,
+      page: 1,
+      numToDisplay: 2,
+      sortOrder: 'relevant',
+    };
+  }
+
+  getReviews() {
+
+  }
+
+  filterReviews() {
+
+  }
+
   render() {
     return (
       <div className="reviews-widget">
         <h3>{'Ratings & Reviews'}</h3>
         <div className="reviews-container">
-          <RatingBreakdown />
-          <ReviewList />
+          <RatingBreakdown
+            reviewMetaData={this.props.reviewMetaData}
+            reviewAverage={this.props.reviewAverage}
+            reviewCount={this.state.reviewCount}
```

Comment:
this prop (reviewCount) is undefined further down the tree (RatingsBreakdown, ReviewsSummary, etc). I think this value should be `this.props.reviewCount` not `.state.reviewCount`.

Reply:
yup, thank you!

Reply:
@zrealberg fixed and thank you again!